Libraries/Text/TextInput/Singleline/RCTSinglelineTextInputView.m

```diff
@@ -53,6 +56,19 @@ - (void)setReactBorderInsets:(UIEdgeInsets)reactBorderInsets
   ((RCTUITextField*)self.backedTextInputView).frame = UIEdgeInsetsInsetRect(self.bounds, reactBorderInsets);
   [self setNeedsLayout];
 }
+
+- (void)setUseSecureTextField:(BOOL)useSecureTextField {
```

Comment:
I wonder if this should be backed by a property such that you can compare this to an existing ivar value?

Reply:
Totes